craft_parts/plugins/base.py

```diff
@@ -44,6 +44,11 @@ class Package:
     name: str
     version: str
 
+    def __lt__(self, other: Package) -> bool:
```

Comment:
is this different from setting `order=True` on the dataclass decorator?

Reply:
https://github.com/canonical/craft-parts/pull/943/commits/6a8e8ce17a4a15f57e799bc9b815854bf65c4b24